forward_system/src/system/system.rs

```diff
@@ -41,10 +41,10 @@ impl<O: IOOracle> SystemTypes for ForwardSystemTypes<O> {
 
 impl<O: IOOracle> EthereumLikeTypes for ForwardSystemTypes<O> {}
 
-pub type ForwardRunningSystem<T, PS, TS> = ForwardSystemTypes<ForwardRunningOracle<T, PS, TS>>;
+pub type ForwardRunningSystem = ForwardSystemTypes<ZkEENonDeterminismSource<DummyMemorySource>>;
 
-pub type CallSimulationSystem<T, PS, TS> = ForwardSystemTypes<CallSimulationOracle<T, PS, TS>>;
+pub type CallSimulationSystem = ForwardSystemTypes<ZkEENonDeterminismSource<DummyMemorySource>>;
```

Comment:
Does it make sense to keep these 2 types?

Reply:
It just gets downstreamed, but maybe can remove